app/services/preserved_object_handler.rb

```diff
@@ -163,36 +164,24 @@ def update_version
       end
     end
 
-    handler_results.report_results
+    results.report_results
   end
 
-  private
+  protected
 
-  # TODO: near duplicate of method in catalog_to_moab - extract superclass or moab wrapper class??
-  def moab_validation_errors
-    @moab_errors ||=
-      begin
-        object_dir = "#{storage_location}/#{DruidTools::Druid.new(druid).tree.join('/')}"
-        moab = Moab::StorageObject.new(druid, object_dir)
-        object_validator = Stanford::StorageObjectValidator.new(moab)
-        moab_errors = object_validator.validation_errors(Settings.moab.allow_content_subdirs)
-        @ran_moab_validation = true
-        if moab_errors.any?
-          moab_error_msgs = []
-          moab_errors.each do |error_hash|
-            error_hash.each_value { |msg| moab_error_msgs << msg }
-          end
-          handler_results.add_result(AuditResults::INVALID_MOAB, moab_error_msgs)
-        end
-        moab_errors
-      end
+  def pres_object
+    @pres_object ||= PreservedObject.find_by!(druid: druid)
   end
 
-  # TODO: duplicate of method in catalog_to_moab - extract superclass or moab wrapper class??
-  def ran_moab_validation?
-    @ran_moab_validation ||= false
+  def pres_copy
+    # FIXME: what if there is more than one associated pres_copy?
```

Comment:
same: have we ticketed this missing work yet?

Reply:
that's an old fixme that i just carried over, but just created a ticket:  https://github.com/sul-dlss/preservation_catalog/issues/691